src/libraries/Common/src/System/CodeDom/CodeTypeReference.cs

```diff
@@ -50,7 +50,7 @@ public CodeTypeReference(Type type)
             if (type.IsArray)
             {
                 ArrayRank = type.GetArrayRank();
-                ArrayElementType = new CodeTypeReference(type.GetElementType());
+                ArrayElementType = new CodeTypeReference(type.GetElementType()!);
                 _baseType = null;
             }
             else
```

Comment:
Shouldn't `typeName` be nullable for ctors in line 71 and 76 given that `Initialize` can take a nullable `typeName`?

Reply:
For this assembly, these types are not public and we don't ever pass `null` into `typeName`. Since this type is not public in this assembly, but is for System.CodeDom, I didn't want to go further than what was necessary to get nullable enabled for S.P.DCS. When/if System.CodeDom gets annotated for nullable, that person will need to decide the public nullability of this class.